ldes-server-port-ingest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/extractor/MemberModelExtractorTest.java

```diff
@@ -20,19 +19,6 @@
 
 class MemberModelExtractorTest {
 
-    @Test
-    void test_ExtractNamedSubjects() {
-        final List<Resource> expectedNamedNodes = Stream.of("bart", "lisa", "homer")
-                .map("http://temporary.org#%s"::formatted)
-                .map(ResourceFactory::createResource)
-                .toList();
-        final Model model = RDFParser.source("bulk-members/simpsons/all.nq").lang(Lang.NQ).toModel();
-
-        final List<Resource> namedNodes = MemberModelExtractor.initialize(model).getNamedSubjectNodes();
-
-        assertThat(namedNodes).containsExactlyInAnyOrderElementsOf(expectedNamedNodes);
-    }
```

Comment:
can't you reuse this for excactAllMemberModels method? since you merged it into that method?